### PR TITLE
Adding bullseye to layouts. Fixing grid gutters.

### PR DESCRIPTION
**Pull request by markcaron**

### CODE REVIEW

**Comment by markcaron** [comment]
Updated Storybook file.

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "postinstall": "npm run bootstrap && npm run wct-kludge",
     "precommit": "npm run update-test-suite > /dev/null && lint-staged",
     "update-test-suite": "node scripts/test-suite-inject.js",
-    "storybook": "start-storybook -c .storybook -p 9001",
```
Comment (mwcz) [inline package.json]: Why was the reason for this change?

File: elements/rh-number/rh-number.story.js [hunk 2]
```diff
@@ -1,94 +1,94 @@
-import { storiesOf } from "@storybook/polymer";
```
Comment (mwcz) [inline elements/rh-number/rh-number.story.js]: I'm guess this was done back before the third-party dependencies issue was figured out.  That is resolved now, so if you merge master into your PR branch you shouldn't have to comment out rh-number's story in order to get storybook working.

**Comment by mwcz** [comment]
Closing for now.

**Comment by kylebuch8** [comment]
@markcaron - Do you have the bullseye Sass file handy? I renamed this element to rh-layouts and there's a bunch of merge conflicts now. It might be easier to close this PR and just add it back in again and submit another PR.

**Comment by markcaron** [comment]
@kylebuch8, I should. I'll add it back and open a new PR.

**Pull request closed without merging**